client/components/Username.vue

```diff
@@ -72,7 +74,10 @@ export default defineComponent({
 
 		const store = useStore();
 
+		const modeName = constants.modeCharToName;
```

Comment:
modeName seems a poor name, why not keep it as modeCharToName?
It's a mapping, not a mode name.

Reply:
That's fair, I'll have it changed